tool/cmd/cmd-go.go

```diff
@@ -0,0 +1,45 @@
+// Copyright The OpenTelemetry Authors
```

Comment:
Actually, I don't have a preference. As long as we keep the style consistent, I'm happy to change everything to the if short statement form later on!